### PR TITLE
tahoe-lafs: 1.13.0 -> 2021-07-09

### PR DESCRIPTION
**Pull request by aaronjanse**
###### Motivation for this change

See https://github.com/tahoe-lafs/community/issues/10. Tahoe-LAFS would like to use NixOS for a demo deployment but the current derivation is broken. The next step is testing and updating the NixOS module to allow arbitrary settings.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by siraben** [comment]
@aaronjanse please fix the merge conflict

**Comment by aaronjanse** [comment]
Oh awesome, it looks like I can remove most of the non-tahoe-lafs commits. I'm surprised that I didn't notice the merge conflict while submitting the PR through the Github UI

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 132197` at 1c909de5 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>tahoe-lafs</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `fetchurl`.
Near pkgs/tools/networking/p2p/tahoe-lafs/default.nix:1:3:
```
  |
1 | { fetchurl, lib, nettools, python3Packages, texinfo, fetchFromGitHub }:
  |   ^
```
</li>
  </ul>
</details>

**Comment by aaronjanse** [comment]
I'll make sure to remove the unused argument in the morning

File: pkgs/tools/networking/p2p/tahoe-lafs/default.nix [hunk 1]
```diff
@@ -1,18 +1,20 @@
-{ fetchurl, lib, nettools, pythonPackages, texinfo }:
+{ fetchurl, lib, nettools, python3Packages, texinfo, fetchFromGitHub }:
```
Comment (Mic92) [inline pkgs/tools/networking/p2p/tahoe-lafs/default.nix:1]:
```suggestion
{ lib, nettools, python3Packages, texinfo, fetchFromGitHub }:
```

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 132197` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tahoe-lafs</li>
  </ul>
</details>

File: pkgs/tools/networking/p2p/tahoe-lafs/default.nix (hunk 1 above)
Comment (aaronjanse) [reply pkgs/tools/networking/p2p/tahoe-lafs/default.nix]: Thanks!

**Pull request merged**